velox/docs/functions/spark/array.rst

```diff
@@ -196,6 +196,24 @@ Array Functions
         SELECT size(NULL, true); -- -1
         SELECT size(NULL, false); -- NULL
 
+.. spark:function:: slice(array(E), start, length) -> array(E)
+
+    Returns a subarray starting at 1-based index ``start`` or from end if negative, with ``length`` elements.
+
```

Comment:
nit: remove the empty lines between sentences.

Reply:
If remove the empty line, sphinx can't render it correctly. Can see https://github.com/facebookincubator/velox/pull/9620#discussion_r1698269038

Reply:
I tried to remove the empty lines locally and it seems render correctly. Could you confirm?

![image](https://github.com/user-attachments/assets/2638e602-c3b7-4594-8856-49959f24ea01)

Reply:
I thought we want to separate it in lines instead of collapse them, like this:
![image](https://github.com/user-attachments/assets/99dff558-968d-4643-8bef-839299a51936)

Will change here to make them to in 1 line.

Reply:
I think we'd better the style of other documentation. Seeing: 

https://github.com/facebookincubator/velox/blob/9dda4c68477fc64caa9fb51174e46f3c36b519ea/velox/docs/functions/spark/array.rst?plain=1#L59-L62

Thanks.